operations/operations/publishing/publishing-guidelines/documentation-style-guide.md

```diff
@@ -1,18 +1,14 @@
 # Documentation Style Guide
 
-When writing for Mattermost, always think about the audience that will be reading your writing. Mattermost’s 
-audience can vary, but we typically communicate with a technical audience and want to present Mattermost as 
-being a friend of the practitioner in development, IT, and operations. It must be accurate and clear, be 
-presented with a style and tone that is appropriate for technical content, and easy to navigate.
+When writing for Mattermost, always think about the audience that will be reading your writing. Mattermost’s audience can vary, but we typically communicate with a technical audience and want to present Mattermost as being a friend of the practitioner in development, IT, and operations. It must be accurate and clear, be presented with a style and tone that is appropriate for technical or educational content, and easy to navigate.
+
+When writing educational content the audience is placed at the center of attention, opposing technical documentation where it is the documents content.
 
 ## Document structure
 
-Structure and organization are an important part of a document's ease of use and its understandability. 
-Information should be organized and presented in a logical order, with similar subjects grouped together in the 
-same section. In most cases, a document has a title, an introductory paragraph, and one or more sections.
+Structure and organization are an important part of a document's ease of use and its understandability. Information should be organized and presented in a logical order, with similar subjects grouped together in the same section. In most cases, a document has a title, an introductory paragraph, and one or more sections.
 
-Try to keep only one topic in a page. Shorter topics are easier to reuse in other documents, are 
-easier to write and edit, and are easier to translate.
+Try to keep only one topic in a page, with the exception of educational guides. Shorter topics are easier to reuse in other documents, are easier to write and edit, and are easier to translate.
```

Comment:
```suggestion
Try to keep only one topic in a page. Shorter topics are easier to reuse in other documents, are easier to write and edit, and are easier to translate. 

An exception to this is Customer Education content which covers multiple topics per page.
```
Maybe have a link to a single-topic technical doc and then one to a CE doc for reference?

Reply:
@justinegeffen  I'd like to keep references to a minimum for maintainability purposes and I don't think examples are needed, as all CE content (right now there is only one guide, the Mobile App Deployment Guide) covers multiple topics on a page.